### PR TITLE
feat(i18n): 🌐 Portuguese translation

### PR DESCRIPTION
**Pull request by miguelfferraz**
This PR implements Portuguese translation for lazydocker.

I hope you enjoy this contribution!

image

### CODE REVIEW

File: pkg/i18n/portuguese.go [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+		ErrorOccurred:                     "Um erro ocorreu! Por favor, crie uma issue em https://github.com/jesseduffield/lazydocker/issues",
+		ConnectionFailed:                  "Falha na conexão com o cliente Docker. Você pode precisar reiniciar o seu cliente Docker",
+		UnattachableContainerError:        "O contêiner não suporta anexação. Você deve executar o serviço com a flag '-it' ou usar `stdin_open: true, tty: true` no arquivo docker-compose.yml",
+		WaitingForContainerInfo:    	   "Não é possível prosseguir até que o Docker forneça mais informações sobre o contêiner. Por favor, tente novamente em alguns momentos.",
+		CannotAttachStoppedContainerError: "Você não pode anexar a um contêiner parado, você precisa iniciá-lo primeiro (o que você pode fazer com a tecla 'r') (sim, sou preguiçoso demais para fazer isso automaticamente para você) (aliás, bem legal que eu posso me comunicar diretamente com você na forma de uma mensagem de erro)",
+		CannotAccessDockerSocketError:     "Não é possível acessar o sôquete docker em: unix:///var/run/docker.sock\nExecute o lazydoccker como root ou leia https://docs.docker.com/install/linux/linux-postinstall/",
```
Comment (jesseduffield) [inline pkg/i18n/portuguese.go:24]:
```suggestion
		CannotAccessDockerSocketError:     "Não é possível acessar o sôquete docker em: unix:///var/run/docker.sock\nExecute o lazydocker como root ou leia https://docs.docker.com/install/linux/linux-postinstall/",
```

**Comment by miguelfferraz** [comment]
@jesseduffield

**Comment by jesseduffield** [comment]
Looks like there's a linting issue @miguelfferraz

**Comment by jesseduffield** [comment]
Nice work @miguelfferraz !

**Pull request merged**